shared-data/labware/schemas/3.json

```
@@ -615,31 +614,42 @@
         }
       }
     },
-    "locatingFeaturesAsParent": {
-      "type": "object",
-      "description": "Explict locating features when this labware acts as the parent in a stackup.",
-      "required": [],
-      "properties": {}
-    },
-    "locatingFeaturesAsChild": {
-      "type": "object",
-      "description": "Explict locating features when this labware acts as the child in a stackup.",
-      "required": [],
-      "properties": {}
-    },
     "extents": {
       "description": "The outer dimensions of the labware.",
       "type": "object",
-      "required": ["total", "footprint"],
+      "required": ["total"],
       "additionalProperties": false,
       "properties": {
         "total": {
           "description": "The all-encompassing outer dimensions of the labware.",
           "$ref": "#/definitions/axisAlignedBoundingBox3D"
-        },
-        "footprint": {
-          "description": "The outer dimensions of the labware where it meets the deck.",
-          "$ref": "#/definitions/axisAlignedBoundingBox2D"
+        }
+      }
+    },
+    "features": {
+      "type": "object",
+      "description": "Locating features when this labware is a part of a stackup.",
+      "required": ["slotFootprintAsChild"],
+      "properties": {
+        "slotFootprintAsChild": {
+          "description": "The 3D plane of the labware that makes physical contact with a parent entity. Constrained by the dimensions of a slot.",
+          "type": "object",
+          "required": ["backLeft", "frontRight", "z"],
+          "additionalProperties": false,
+          "properties": {
+            "backLeft": {
+              "description": "(-x, +y) relative to the labware origin.",
+              "$ref": "#/definitions/vector2D"
+            },
+            "frontRight": {
+              "description": "(+x, -y) relative to the labware origin.",
+              "$ref": "#/definitions/vector2D"
+            },
+            "z": {
+              "description": "Always zero. Z-coordinate of the footprint plane.",
```

Nitpicky clarification:

```suggestion
              "description": "Always zero by convention. Z-coordinate of the footprint plane.",
```

Because even though we do want to instruct labware authors to write `z=0`, we don't want to imply to Opentrons software engineers that it's okay to assume `z=0`.